packages/cli/src/main.ts

```diff
@@ -44,6 +45,7 @@ export const main: () => void = () => {
     .command(showProject)
     .command(updateTests)
     .command(uploadBuild)
+    .command(serve)
```

Comment:
Looks like this will hide it from the help menu
```suggestion
    .command("serve", false, serve)
```

Reply:
Oh nice, I'll try that

Reply:
It works -- thanks!